cmd/root.go

```diff
@@ -347,6 +360,16 @@ func parseConfig(cmd *Command, conf *proxy.Config, args []string) error {
 		}
 	}
 
+	if cmd.impersonationChain != "" {
+		accts := strings.Split(cmd.impersonationChain, ",")
```

Comment:
Can this ever be an empty slice?

Reply:
Only if a user passed this: `--impersonate-service-account=,`.